console-extensions.json

```diff
@@ -78,5 +78,12 @@
       "section": "observe",
       "insertAfter": "dashboards"
     }
+  },
+	{
+    "type": "console.action/provider",
+    "properties": {
+      "contextId": "mock-lightspeed", 
```

Comment:
I think this is only a mock for test, but it shouldn't be included in the code

Reply:
Yes, it is only a mock for testing. It is removed in the latest commit.